internal/config/config.go

```diff
@@ -2,9 +2,10 @@ package config
 
 // Options - shortener server configurations
 type Options struct {
-	ServerAddress   string // ServerAddress - address which system use to run shortener server
-	BaseURL         string // BaseURL - host for shortened URLs
+	ServerAddress   string `json:"server_address"` // ServerAddress - address which system use to run shortener server
```

Comment:
It can be optional. 
```suggestion
	ServerAddress   string `json:"server_address,omitempty"` // ServerAddress - address which system use to run shortener server
```